.travis.yml

```diff
@@ -11,6 +11,7 @@ before_install:
   - mkdir -p $GOPATH/src/github.com/vbatts && ln -sf $(pwd) $GOPATH/src/github.com/vbatts/git-validation && go get ./...
 
 before_script:
+  - env
   - echo $TRAVIS_COMMIT_RANGE
```

Comment:
With the `env` (which I approve of), you can drop all the following `echo $TRAVIS_…` lines.